tldw_Server_API/app/core/Chat/streaming_utils.py

```
@@ -16,7 +16,7 @@
 import time
 from collections.abc import AsyncIterator, Iterator
 from datetime import datetime, timezone
-from typing import Any, Optional, Union
+from typing import Any, Callable, Optional, Union
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)